README.md

```diff
@@ -267,6 +275,14 @@ awxJob:
     node: "{{ $labels.node }}"
 ```
 
+Or to limit execution to a host, calculated from the `instance` label:
```

Comment:
Can you remove the 'Or to'? can be written as:
Limit execution to the host calculated from the `instance` label:

If you think this isn't working with the paragraph above you can rephrase it, or leave as is.